docs/integrations/payment-gateways.md

```diff
@@ -43,13 +44,13 @@ This gateway implements payments using [Razorpay](https://razorpay.com/).
 
 Before you start, you need to create your API credentials. Go to your Razorpay account settings, and to [the API Keys section](https://dashboard.razorpay.com/#/app/keys).
 
-| Environment Variable | Description |
+| Plugin Settings | Description |
 | --- | --- |
-| `RAZORPAY_PUBLIC_KEY` | Your Razorpay key id. |
-| `RAZORPAY_SECRET_KEY` | Your Razorpay secret key id. |
-| `RAZORPAY_PREFILL` | Pre-fill the email and customer’s full name if set to `True` (default). |
-| `RAZORPAY_STORE_NAME` | Your store name. |
-| `RAZORPAY_STORE_IMAGE` | An absolute or relative link to your store logo. |
+| `Public API key` | Your Razorpay key id. |
+| `Secret API key` | Your Razorpay secret key id. |
+| `Store customers card` | Determines if Saleor should store cards on payments. |
```

Comment:
This is not supported by the backend. Should we document this setting?

Reply:
@NyanKiyoshi you obviously now more about Razorpay then I do, could you please see if there is a need to update Razopay plugin and its configuration? Maybe we could start with **not** documenting unwanted settings.

Reply:
After going through their new documentation, looks like it is a usable feature. Maybe we could just add a note that it not currently possible then? That way would be fine.

Reply:
@MonikaMrozek can we just add a note here with this information?

Reply:
Not sure what the note should read, guys? What is not currently possible?